sdk/samples/iot/aziot_esp32/aziot_esp32.ino

```diff
@@ -230,7 +230,7 @@ static void getTelemetryPayload(az_span payload, az_span* out_payload)
   az_span original_payload = payload;
 
   payload = az_span_copy(
-      payload, AZ_SPAN_FROM_STR("{ \"deviceId\": \"" IOT_CONFIG_DEVICE_ID "\", \"msgCount\": "));
+      payload, AZ_SPAN_FROM_STR("{ \"msgCount\": "));
```

Comment:
Happy to see you removed the device ID from the payload since this is already sent via message properties anyway! Saving a couple of bytes on the wire is always a good idea. Thank you!